python/lsst/obs/base/ingest_tests.py

```diff
@@ -110,92 +112,26 @@ def testCopy(self):
 
     def testHardLink(self):
         self.config.transfer = "hardlink"
-        self.runIngestTest()
+        try:
+            self.runIngestTest()
+        except PermissionError as err:
+            raise unittest.SkipTest("Skipping hard-link test because input data"
+                                    " is on a different filesystem.") from err
 
     def testInPlace(self):
         """Test that files already in the directory can be added to the
         registry in-place.
         """
-        # hardlink into repo root manually
+        # copy into repo root manually
```

Comment:
If you have a problem with testdata being in a different place to the test can't we use `os.symlink` instead of copying all the data across?

Reply:
Yeah, I think that should work; I'll give it a try.  Wouldn't help with the hardlink test, but I've just skipped that one anyway.